### ISSUE TITLE
snapserver not forwarding to client

### ISSUE THREAD

**Issue by wyllys66** [badaix/snapcast#1286]
I have latest snapserver and snapclient running on an RPI5 running debian bookworm.

They work great when receiving data from a remote server like Music Assistant, the server gets the data and plays it to the client.

I also have Shairport-sync on the same server configured to write to a local pipe and snapserver is configured to read from that pipe as a source.

The server logs when the shairport stream starts and stops but it never sends any data to the snapclient and I cannot figure out why it is not working.  Any suggestions?

snapserver v0.26.0
snapclient v0.26.0

Server log:
```log
Sep 07 12:38:17 rpi5music snapserver[24768]: State changed: Shairport, state: playing => idle
Sep 07 12:38:17 rpi5music snapserver[24768]: onStateChanged (Shairport): idle
Sep 07 12:38:19 rpi5music snapserver[24768]: State changed: Shairport, state: idle => playing
Sep 07 12:38:19 rpi5music snapserver[24768]: onStateChanged (Shairport): playing

```

client log:
```log
Sep 07 12:33:10 rpi5music snapclient[8743]: Codec: flac, sampleformat: 48000:16:2
Sep 07 12:33:10 rpi5music snapclient[8743]: Player name: alsa, device: default, description: Default Audio Device, idx: 1, sharing mode: unspecified, parameters: <none>
Sep 07 12:33:10 rpi5music snapclient[8743]: Mixer mode: software, parameters: <none>
Sep 07 12:33:10 rpi5music snapclient[8743]: Sampleformat: 48000:16:2, stream: 48000:16:2
Sep 07 12:33:10 rpi5music snapclient[8743]: Using default buffer_time: 80 ms, default fragments: 4
Sep 07 12:33:10 rpi5music snapclient[8743]: PCM name: default, sample rate: 48000 Hz, channels: 2, buffer time: 80000 us, periods: 4, period time: 20000 us, period frames: 960
Sep 07 12:33:10 rpi5music snapclient[8743]: No chunks available
Sep 07 12:33:10 rpi5music snapclient[8743]: Failed to get chunk
Sep 07 12:33:10 rpi5music snapclient[8743]: diff to server [ms]: 0.005
Sep 07 12:33:15 rpi5music snapclient[8743]: No chunk received for 5000ms. Closing ALSA.
```

**Comment by J1810Z** [comment]
I am running into exactly the same issue. How did you resolve it?

**Comment by wyllys66** [comment]
By not using snapcast at all.   All I really wanted was to stream from my phone to my older stereo receiver (which doesnt have bluetooth).  I wanted to get rid of my Sonos Connect (S1), so after tinkering with various combinations of snapcast and shairport-sync, I ended up just needing shairport-sync by itself.  Now I can easily airplay music from my other Apple devices directly to the Pi running shairport-sync with direct audio output jacks into the stereo.  

The snapserver/snapclient worked "mostly" but there were too many gaps and pauses (probably due to my home wifi network)  so I just gave up on that.
